src/screens/ChatScreen/AnonymousChatScreen.tsx

```diff
@@ -135,7 +135,7 @@ const AnonymousChatScreen = () => {
       />
       <View style={styles.inputContainer}>
         <InputMessageV2
-          onSendButtonClicked={sendChat}
+          onSendButtonClicked={sendChatMutation.mutate}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change reflects the shift from calling the `sendChat` function directly to using the `mutate` method on `sendChatMutation`. This is part of the transition to using react-query's mutation functions, which provide additional features like automatic retries and error handling.

```diff
-          onSendButtonClicked={sendChat}
+          onSendButtonClicked={sendChatMutation.mutate}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->